features/step_definitions/collection_steps.rb

```diff
@@ -212,21 +212,26 @@
   page.should have_content("Anonymous [#{authors}]")
 end
 
+# Scenarios using this step need the @javascript tag
 Then /^the author of "([^\"]*)" should be publicly visible$/ do |title|
   work = Work.find_by(title: title)
+  byline = work.users.first.pseuds.first.byline
   visit work_path(work)
-  page.should have_content("by <a href=\"#{user_url(work.users.first)}\"><strong>#{work.users.first.pseuds.first.byline}")
+  step %{I should see "#{byline}" within "title"}
+  step %{I should see "#{byline}" within ".byline"}
   if work.collections.first
     visit collection_path(work.collections.first)
-    page.should have_content("#{title} by #{work.users.first.pseuds.first.byline}")
+    page.should have_content("#{title} by #{byline}")
   end
 end
 
+# Scenarios using this step need the @javascript tag
```

Comment:
And here, too?

Reply:
yep, resolved in 186364e72d5a1338b46a584378e481f0b2142afb